apps/chat/src/components/Common/ItemContextMenu.tsx

```diff
@@ -268,26 +266,11 @@ export default function ItemContextMenu({
         onClick: onPublish,
         disabled: disableAll,
       },
-      {
-        name: t('Update'),
```

Comment:
Don't we need `Update` option? It's like `Publish` but `published by me`

Reply:
reverted with TODO comment